README.md

```diff
@@ -202,6 +202,37 @@ var body = new BodyBuilder()
 // }
 ```
 
+#### Filters Aggregations
+To add a filters aggregation, the second argument expects a callback. You will be passed a filters-builder with which you can build filters just like you normally do with BodyBuilder. The first argument is a required filter name. The last argument here is also the callback for nested aggregations.
+
+```js
+var body = new BodyBuilder()
+  .aggregation('filters', filtersBuilder => {
+    return filtersBuilder.filter('colors','term', 'color', 'red')
+                         .filter('statuses', 'term', 'status', 'success')
+  }, 'products', agg => agg.aggregation('avg', 'price', 'avg_price'))
+  .build()
+// body == {
+//   "aggregations": {
+//     "products": {
+//       "filters": {
+//         "filters": {
+//           "colors": { "term": { "color": "red" } },
+//           "statuses": { "term": { "status": "success" } }
+//         }
+//       },
+//       "aggs": {
+//         "avg_price": {
+//           "avg": {
+//             "field": "price"
+//           }
+//         }
+//       }
+//     }
+//   }
+// }
+```
+
```

Comment:
Instead of duplicating this section of the readme which is very similar to `Filter aggregations`, maybe it makes sense to combine the two sections into one and show both examples.